examples/example.hpp

```diff
@@ -1082,9 +1082,8 @@ std::string depth_with_stream_type_present(rs2_stream type)
             for (auto profile : sensor.get_stream_profiles())
             {
                 if (profile.stream_type() == type)
-                    return dev.get_info(RS2_CAMERA_INFO_SERIAL_NUMBER);
+                    out_serial = dev.get_info(RS2_CAMERA_INFO_SERIAL_NUMBER);
```

Comment:
Check for `dev.supports(RS2_CAMERA_INFO_SERIAL_NUMBER)` before the attribute query